### ISSUE TITLE
Resolved player names in /tp and /tphere commands not always accurate

### ISSUE THREAD

**Issue by Ceddix** [WiIIiam278/HuskHomes2#243]
If you teleport to a player with HuskHomes but don't use the full player name (or other case sensitivity), the output uses the name you entered (e.g. `Ceddix` only `ced`). It might make more sense if the full name (i.e. `Ceddix`) is still displayed in the message.

**Comment by WiIIiam278** [comment]
which commands specifically do this? and are you running v3.0.5 (dev builds)

**Comment by Ceddix** [comment]
I'll try with dev builds, one second

**Comment by Ceddix** [comment]
Actually now it won't load at all haha
```
[14:10:18] [Server thread/INFO]: [HuskHomes] Enabling HuskHomes v3.0.5-2d8ed4f
[14:10:18] [Server thread/INFO]: [HuskHomes] Loading plugin configuration settings & locales...
[14:10:18] [ForkJoinPool.commonPool-worker-1/ERROR]: [HuskHomes] Failed to load data from the config
java.util.concurrent.CompletionException: could not determine a constructor for the tag tag:yaml.org,2002:net.william278.huskhomes.config.Settings$ConnectionPoolOptions
 in reader, line 20, column 22:
        connection_pool: !!net.william278.huskhomes.confi ... 
                         ^

	at java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:315) ~[?:?]
	at java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:320) ~[?:?]
	at java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1770) ~[?:?]
	at java.util.concurrent.CompletableFuture$AsyncSupply.exec(CompletableFuture.java:1760) ~[?:?]
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373) ~[?:?]
	at java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182) ~[?:?]
	at java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655) ~[?:?]
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622) ~[?:?]
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165) ~[?:?]
Caused by: net.william278.huskhomes.libraries.boostedyaml.libs.org.snakeyaml.engine.v2.exceptions.ConstructorException: could not determine a constructor for the tag tag:yaml.org,2002:net.william278.huskhomes.config.Settings$ConnectionPoolOptions
 in reader, line 20, column 22:
        connection_pool: !!net.william278.huskhomes.confi ... 
                         ^

	at net.william278.huskhomes.libraries.boostedyaml.libs.org.snakeyaml.engine.v2.constructor.BaseConstructor.lambda$constructObjectNoCheck$0(BaseConstructor.java:138) ~[HuskHomes.jar:?]
	at java.util.Optional.orElseThrow(Optional.java:403) ~[?:?]
	at net.william278.huskhomes.libraries.boostedyaml.libs.org.snakeyaml.engine.v2.constructor.BaseConstructor.constructObjectNoCheck(BaseConstructor.java:136) ~[HuskHomes.jar:?]
	at net.william278.huskhomes.libraries.boostedyaml.engine.ExtendedConstructor.constructObjectNoCheck(ExtendedConstructor.java:66) ~[HuskHomes.jar:?]
	at net.william278.huskhomes.libraries.boostedyaml.libs.org.snakeyaml.engine.v2.constructor.BaseConstructor.constructObject(BaseConstructor.java:127) ~[HuskHomes.jar:?]
	at net.william278.huskhomes.libraries.boostedyaml.libs.org.snakeyaml.engine.v2.constructor.BaseConstructor.constructMapping2ndStep(BaseConstructor.java:237) ~[HuskHomes.jar:?]
	at net.william278.huskhomes.libraries.boostedyaml.libs.org.snakeyaml.engine.v2.constructor.StandardConstructor.constructMapping2ndStep(StandardConstructor.java:122) ~[HuskHomes.jar:?]
	at net.william278.huskhomes.libraries.boostedyaml.libs.org.snakeyaml.engine.v2.constructor.BaseConstructor.constructMapping(BaseConstructor.java:218) ~[HuskHomes.jar:?]
	at net.william278.huskhomes.libraries.boostedyaml.libs.org.snakeyaml.engine.v2.constructor.StandardConstructor$ConstructYamlMap.construct(StandardConstructor.java:308) ~[HuskHomes.jar:?]
	at net.william278.huskhomes.libraries.boostedyaml.engine.ExtendedConstructor$ConstructMap.construct(ExtendedConstructor.java:114) ~[HuskHomes.jar:?]
	at net.william278.huskhomes.libraries.boostedyaml.libs.org.snakeyaml.engine.v2.constructor.BaseConstructor.constructObjectNoCheck(BaseConstructor.java:140) ~[HuskHomes.jar:?]
	at net.william278.huskhomes.libraries.boostedyaml.engine.ExtendedConstructor.constructObjectNoCheck(ExtendedConstructor.java:66) ~[HuskHomes.jar:?]
	at net.william278.huskhomes.libraries.boostedyaml.libs.org.snakeyaml.engine.v2.constructor.BaseConstructor.constructObject(BaseConstructor.java:127) ~[HuskHomes.jar:?]
	at net.william278.huskhomes.libraries.boostedyaml.libs.org.snakeyaml.engine.v2.constructor.BaseConstructor.constructMapping2ndStep(BaseConstructor.java:237) ~[HuskHomes.jar:?]
	at net.william278.huskhomes.libraries.boostedyaml.libs.org.snakeyaml.engine.v2.constructor.StandardConstructor.constructMapping2ndStep(StandardConstructor.java:122) ~[HuskHomes.jar:?]
	at net.william278.huskhomes.libraries.boostedyaml.libs.org.snakeyaml.engine.v2.constructor.BaseConstructor.constructMapping(BaseConstructor.java:218) ~[HuskHomes.jar:?]
	at net.william278.huskhomes.libraries.boostedyaml.libs.org.snakeyaml.engine.v2.constructor.StandardConstructor$ConstructYamlMap.construct(StandardConstructor.java:308) ~[HuskHomes.jar:?]
	at net.william278.huskhomes.libraries.boostedyaml.engine.ExtendedConstructor$ConstructMap.construct(ExtendedConstructor.java:114) ~[HuskHomes.jar:?]
	at net.william278.huskhomes.libraries.boostedyaml.libs.org.snakeyaml.engine.v2.constructor.BaseConstructor.constructObjectNoCheck(BaseConstructor.java:140) ~[HuskHomes.jar:?]
	at net.william278.huskhomes.libraries.boostedyaml.engine.ExtendedConstructor.constructObjectNoCheck(ExtendedConstructor.java:66) ~[HuskHomes.jar:?]
	at net.william278.huskhomes.libraries.boostedyaml.libs.org.snakeyaml.engine.v2.constructor.BaseConstructor.constructObject(BaseConstructor.java:127) ~[HuskHomes.jar:?]
	at net.william278.huskhomes.libraries.boostedyaml.libs.org.snakeyaml.engine.v2.constructor.BaseConstructor.constructMapping2ndStep(BaseConstructor.java:237) ~[HuskHomes.jar:?]
	at net.william278.huskhomes.libraries.boostedyaml.libs.org.snakeyaml.engine.v2.constructor.StandardConstructor.constructMapping2ndStep(StandardConstructor.java:122) ~[HuskHomes.jar:?]
	at net.william278.huskhomes.libraries.boostedyaml.libs.org.snakeyaml.engine.v2.constructor.BaseConstructor.constructMapping(BaseConstructor.java:218) ~[HuskHomes.jar:?]
	at net.william278.huskhomes.libraries.boostedyaml.libs.org.snakeyaml.engine.v2.constructor.StandardConstructor$ConstructYamlMap.construct(StandardConstructor.java:308) ~[HuskHomes.jar:?]
	at net.william278.huskhomes.libraries.boostedyaml.engine.ExtendedConstructor$ConstructMap.construct(ExtendedConstructor.java:114) ~[HuskHomes.jar:?]
	at net.william278.huskhomes.libraries.boostedyaml.libs.org.snakeyaml.engine.v2.constructor.BaseConstructor.constructObjectNoCheck(BaseConstructor.java:140) ~[HuskHomes.jar:?]
	at net.william278.huskhomes.libraries.boostedyaml.engine.ExtendedConstructor.constructObjectNoCheck(ExtendedConstructor.java:66) ~[HuskHomes.jar:?]
	at net.william278.huskhomes.libraries.boostedyaml.libs.org.snakeyaml.engine.v2.constructor.BaseConstructor.constructObject(BaseConstructor.java:127) ~[HuskHomes.jar:?]
	at net.william278.huskhomes.libraries.boostedyaml.libs.org.snakeyaml.engine.v2.constructor.BaseConstructor.construct(BaseConstructor.java:86) ~[HuskHomes.jar:?]
	at net.william278.huskhomes.libraries.boostedyaml.engine.ExtendedConstructor.construct(ExtendedConstructor.java:56) ~[HuskHomes.jar:?]
	at net.william278.huskhomes.libraries.boostedyaml.libs.org.snakeyaml.engine.v2.constructor.BaseConstructor.constructSingleDocument(BaseConstructor.java:73) ~[HuskHomes.jar:?]
	at net.william278.huskhomes.libraries.boostedyaml.YamlDocument.reload(YamlDocument.java:259) ~[HuskHomes.jar:?]
	at net.william278.huskhomes.libraries.boostedyaml.YamlDocument.reload(YamlDocument.java:222) ~[HuskHomes.jar:?]
	at net.william278.huskhomes.libraries.boostedyaml.YamlDocument.<init>(YamlDocument.java:91) ~[HuskHomes.jar:?]
	at net.william278.huskhomes.libraries.boostedyaml.YamlDocument.create(YamlDocument.java:773) ~[HuskHomes.jar:?]
	at net.william278.huskhomes.libraries.boostedyaml.YamlDocument.create(YamlDocument.java:792) ~[HuskHomes.jar:?]
	at net.william278.huskhomes.libraries.annotaml.YamlObjectMap.parse(YamlObjectMap.java:70) ~[HuskHomes.jar:?]
	at net.william278.huskhomes.libraries.annotaml.Annotaml.create(Annotaml.java:121) ~[HuskHomes.jar:?]
	at net.william278.huskhomes.libraries.annotaml.Annotaml.create(Annotaml.java:89) ~[HuskHomes.jar:?]
	at net.william278.huskhomes.BukkitHuskHomes.lambda$reload$20(BukkitHuskHomes.java:522) ~[HuskHomes.jar:?]
	at java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768) ~[?:?]
	... 6 more
[14:10:18] [Server thread/ERROR]: [HuskHomes] Failed to load plugin configuration settings and/or locales
[14:10:18] [Server thread/ERROR]: [HuskHomes] Failed to initialize HuskHomes. The plugin will now be disabled
[14:10:18] [Server thread/INFO]: [HuskHomes] Disabling HuskHomes v3.0.5-2d8ed4f
```

**Comment by WiIIiam278** [comment]
Yeah that's expected. Regen config

**Comment by Ceddix** [comment]
Okay, I updated and the it does this on `/tp` and `/tphere` command. Not on `/tpa` or `/tpahere` _(I didn't test the other commands)_.

**Comment by WiIIiam278** [comment]
Yeah ok sounds about right. This is a bug as it should display the name.

**Comment by WiIIiam278** [comment]
Fixed in [4164f07](https://github.com/WiIIiam278/HuskHomes2/commit/4164f0784172ed89b91567e58f12d7899c155662)
